src/entities/dto/api-key/update-api-key.dto.ts

```diff
@@ -0,0 +1,21 @@
+import { ApiProperty } from "@nestjs/swagger";
```

Comment:
This dto looks very similar to "CreateApiKeyDto". Is there need for two seperate classes :-)?